plexus.config.development

```diff
@@ -7,3 +7,6 @@ PLEXUS_TLS_KEY_FILE=dev/tls.key
 PLEXUS_MESH_CENTRAL_URL=ws://localhost:8086
 PLEXUS_MESH_CENTRAL_USER=plexus
 PLEXUS_MESH_CENTRAL_PASS=plexus
+
+PLEXUS_AUTH_USER="admin"
+PLEXUS_AUTH_PASSWORD="foobaz"
```

Comment:
I think for development, it is simpler if we omit the credentials in here, then we won't have to remember them, when creating a session.

Reply:
Yes I agree